kornia/geometry/calibration/distort.py

```diff
@@ -0,0 +1,128 @@
+import torch
+
+
+# Based on https://github.com/opencv/opencv/blob/master/modules/calib3d/src/distortion_model.hpp#L75
+def tiltProjection(taux: torch.Tensor, tauy: torch.Tensor, inv: bool = False) -> torch.Tensor:
+    r"""Estimate the tilt projection matrix or the inverse tilt projection matrix
+
+    Args:
+        taux (torch.Tensor): Rotation angle in radians around the :math:`x`-axis with shape :math:`(*, 1)`.
```

Comment:
```suggestion
        taux: Rotation angle in radians around the :math:`x`-axis with shape :math:`(*, 1)`.
```
type in docs automatically injected. remove below too